tests/general/InvalidLinkBearTest.py

```diff
@@ -90,6 +90,7 @@ def test_run(self):
         [httpbin](http://httpbin.org/status/200)
         |http://httpbin.org/status/200|
         <h3>Something http://httpbin.org/status/200</h3>
+        repo=\\"http://httpbin.org/status/200\\"
```

Comment:
Ah jeez, right. But then shouldn't there be a test verifying the backslash part?